src/components/prediction/OrgUnitTree.tsx

```diff
@@ -3,32 +3,39 @@ import i18n from "@dhis2/d2-i18n";
 import { useEffect } from "react";
 import { OrganisationUnitTree } from "@dhis2/ui";
 import useOrgUnitRoots from "../../hooks/useOrgUnitRoots";
+import React from "react";
```

Comment:
I don't think this import is needed.

Reply:
I think TypeScript requires it, get an error if I remove it

Reply:
Ok, could be added as:

`import React, { useEffect } from "react";`

For DHIS2 projects we include external imports before internal ones.